content/house/_index.md

```diff
@@ -13,18 +13,16 @@ Das im traditionellen Stil restaurierte, rustikale Olivenbauernhäuschen hat sei
 
 Das erste Schlafzimmer: 
 
-!['Das erste Schlafzimmer'](/static/images/bett.jpeg) !['Das erste Schlafzimmer aus einer anderen Perspektive'](/static/images/bett2.jpeg)
+!['Das erste Schlafzimmer'](/static/images/bett.jpeg)
+<!--- !['Das erste Schlafzimmer aus einer anderen Perspektive'](/static/images/bett2.jpeg) --->
```

Comment:
usually comments are not well perceived in production code. if you don't need this information, remove it.